conf/jetty/jetty.xml.production

```diff
@@ -639,17 +630,17 @@
         <Call name="addRule">
           <Arg>
             <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
-              <Set name="pattern">zimbrax/.*/fonts/*</Set>
+              <Set name="pattern">/zimbrax/.*/fonts/*</Set>
               <Set name="name">Cache-Control</Set>
               <Set name="value">Max-Age=31536000,public</Set>
             </New>
           </Arg>
         </Call>
-        
+
         <Call name="addRule">
           <Arg>
             <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
-              <Set name="pattern">zimbrax/.*/assets/*</Set>
+              <Set name="pattern">/zimbrax/.*/assets/*</Set>
```

Comment:
don't we need %%zimbraMailURL%% here?

Reply:
pattern having /zimbrax in the url let it be /zimbra/zimbrax . or only /zimbra all pattern are obeyed  by above rule so caching rules are getting applied